internal/command/deploy/machines_launchinput.go

```diff
@@ -47,6 +48,8 @@ func (md *machineDeployment) launchInputForLaunch(processGroup string, guest *ap
 		mConfig.Standbys = standbyFor
 	}
 
+	mConfig.Files = md.files
```

Comment:
Shouldn't files come from fly.toml?  I buy it could be useful to override them using flags on `fly deploy` but generally speaking you want to declare them in the app configuration file. do you think?

Reply:
I wasn't sure TBH so figured I'd start with flags and see what others thought.